scheduler/src/test/scala/com/sky/kms/e2e/package.scala

```diff
@@ -8,7 +8,7 @@ import scala.concurrent.duration._
 
 package object e2e {
 
-  val Tolerance = 400 millis
+  val Tolerance = 1300 millis
```

Comment:
It fails on my laptop as well... perhaps we could redefine this in terms of PatienceConfig in `SchedulerIntSpecBase`.